### DISCUSSION TITLE
Welcome to Matlab-GAN Discussions!

### DISCUSSION THREAD

**Discussion by zcemycl** [zcemycl/Matlab-GAN#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by smalllong123** [comment]
Why is full connection layer rather than convolution layer used for images？

**Reply by zcemycl** [reply]
which gan model do you refer to?

**Reply by smalllong123** [reply]
The most basic GAN model

**Reply by zcemycl** [reply]
Mnist is a small dataset with small images. Also, people start learning neural network through fully connected layers.

**Reply by smalllong123** [reply]
thank you very much

**Reply by zcemycl** [reply]
Please star the repo if you want to support this kind of discussion.
